project/scripts/cmdTests

```diff
@@ -18,6 +18,10 @@ echo "testing sbt dotc -decompile"
 "$SBT" ";dotc -decompile -color:never -classpath $OUT $MAIN" > "$tmp"
 grep -qe "def main(args: scala.Array\[scala.Predef.String\]): scala.Unit =" "$tmp"
 
+echo "testing sbt dotc -decompile from file"
+"$SBT" ";dotc -decompile -color:never -classpath $OUT $OUT/$TASTY" #> "$tmp"
```

Comment:
I don't get it. You're supposed to output to `tmp`. Why is it commented out?

Reply:
I was debugging and forgot to reenable it

Reply:
Now it is uncommented